router/src/main/kotlin/io/moia/router/RequestHandler.kt

```diff
@@ -126,73 +146,71 @@ abstract class RequestHandler : RequestHandler<APIGatewayProxyRequestEvent, APIG
 
     private fun createUnprocessableEntityErrorBody(error: UnprocessableEntityError): Any = createUnprocessableEntityErrorBody(listOf(error))
 
-    open fun createApiExceptionErrorResponse(contentType: MediaType, input: APIGatewayProxyRequestEvent, ex: ApiException): APIGatewayProxyResponseEvent =
-        createErrorBody(ex.toApiError()).let {
-            APIGatewayProxyResponseEvent()
-                .withBody(
-                    // in case of 406 we might find no serializer so fall back to the default
-                    if (serializationHandlerChain.supports(contentType, it))
-                        serializationHandlerChain.serialize(contentType, it)
-                    else
-                        serializationHandlerChain.serialize(MediaType.parse(router.defaultContentType), it)
-                )
-                .withStatusCode(ex.httpResponseStatus)
-                .withHeaders(mapOf("Content-Type" to contentType.toString()))
-        }
-
-    open fun createUnexpectedErrorResponse(contentType: MediaType, input: APIGatewayProxyRequestEvent, ex: Exception): APIGatewayProxyResponseEvent =
+    open fun exceptionToResponseEntity(ex: Exception) =
         when (ex) {
-            is JsonParseException -> createResponse(contentType, input,
-                ResponseEntity(422, createUnprocessableEntityErrorBody(
+            is JsonParseException -> ResponseEntity(
+                422,
```

Comment:
Maybe we could use `HttpStatus.SC_UNPROCESSABLE_ENTITY` (from `org.apache.http.HttpStatus`) to increase readability.

Reply:
👍

Reply:
Use constants or even enums would be a good idea. But I would not reuse the constants from Apache HTTP. 
Currently we are not depending on apache http here. And I would rather avoid this dependency just because of the constants.